docs/src/routes/docs/components/checkbox.svx

```diff
@@ -65,7 +65,7 @@ An individual checkbox.
 
 | Name | Default | Type | Description |
 | ---- | ------- | ---- | ----------- |
-| **`value`** | `null` | `string` | The value assigned to the checkbox input. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value) for more information. |
+| **`value`** | `null` | `any` | The value assigned to the checkbox input. Check [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value) for more information. |
```

Comment:
Why? The documentation specifies that this should be a [`DOMString`](https://developer.mozilla.org/en-US/docs/Web/API/DOMString)

Reply:
Yeah, it will be stringified for the DOM, but Svelte doesn't care, and this means that objects can be used as values and retrieved from event values intact

Reply:
Yes, but it will cause a problem with the internal value in the HTML (for example, when using it in a form). The JS code can work fine, but the server will receive a bunch of `[Object object]`s

Reply:
True, but, imo, too bad. The developers should know what they're signing up for when they're using HTML forms, and if they are, ensure that only strings get passed here. Don't see a reason to limit everyone else who doesn't use the HTML forms

Reply:
It's not even a limitation, but just a documentation of the expected property type. We expect it to be a string since we pass it to an element that expects it to be a string. If they wanna act smart and use an object, they should know what they are doing. Otherwise, this will confuse others who know that a `value` has to be a string, and they would try to understand how it can be something else and go through the HTML documentation and a big hassle. Let's just not confuse 99% of the users?